openlibrary/tests/core/test_helpers.py

```diff
@@ -84,10 +84,11 @@ def test_urlsafe():
     assert h.urlsafe("a?") == "a"
 
 def test_get_coverstore_url(monkeypatch):
-    assert h.get_coverstore_url() == "https://covers.openlibrary.org"
-
     from infogami import config
 
+    monkeypatch.delattr(config, "coverstore_url", raising=False)
+    assert h.get_coverstore_url() == "https://covers.openlibrary.org"
```

Comment:
This seems like another good opportunity to move canonical covers service address into a config

Reply:
This test is testing that the fallback (in the absence of config or config file) cover url is `https://covers.openlibrary.org`, so it is taken form config, if it exists, otherwise is set to `https://covers.openlibrary.org`, which seems ok to me, although we should be consistent. Not all config values have defaults and expect the config to be set.

As policy, should we hardcode defaults, or simply reply on config?

Reply:
I think we should rely on config (defaults).

Reply:
This can be a separate issue, merging for now